py_ecc/secp256k1/secp256k1.py

```diff
@@ -12,6 +12,8 @@ def safe_ord(value):
         else:
             return ord(value)
 
+Point2D = Tuple[int, int]
+Point3D = Tuple[int, int, int]
```

Comment:
These could go in a common `py_ecc.types` module so they don't have to be repeated.

Reply:
@pipermerriam Great idea indeed, updated with changes! :+1: